src/streamer.ts

```diff
@@ -31,12 +32,9 @@ export async function startStream(
       continue;
     }
 
-    for (let blockHeight of blockHeights) {
-      const streamerMessage = await fetchStreamerMessage(
-        s3Client,
-        config.s3BucketName,
-        blockHeight
-      );
+    const messages = await Promise.all(
+      blockHeights.map(blockHeight => fetchStreamerMessage(s3Client, config.s3BucketName, blockHeight)));
```

Comment:
Indeed, this is [the strategy we use in the Rust version](https://github.com/near/near-lake-framework/blob/04dddb290a9dc027b2ddf2f58b376e61652022aa/src/lib.rs#L281-L293), but somehow lost this optimization on JS side